resources/js/components/ToggleWhenMobile.vue

```diff
@@ -26,6 +26,12 @@
             buttonLabel() {
                 return this.isOpen ? 'Hide' : 'Show';
             },
+            slotVisible() {
```

Comment:
I think I'd rather evaluate this by adding and removing Tailwind classes instead of evaluating it in JavaScript.

What I imagined was:

```
md:block (so it always shows on md)
hidden (class enabled or disabled based on the isOpen)
```

Reply:
Could you please take a look? It should be resolved now.